BadHabbit/BadHabbit/Kit/DesignKit/Models/StartButtonConstraints.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  StartButtonConstraints.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 25/07/2023.
+//
+
+import Foundation
+
+enum StartButtonConstraints: Int {
```

Comment:
Let's make it more general and rewrite cases as static let.
More general is something like that:
enum LayoutConstants {
static let leadingOffset...
static let trailingOffset...
enum ActionButton {
static let height
static let bottomOffset